packages/connection-form/src/components/connection-form.tsx

```diff
@@ -359,7 +326,7 @@ function ConnectionForm({
   onCancel,
   onAdvancedOptionsToggle,
   openSettingsModal,
-}: ConnectionFormPropsWithoutPreferences): React.ReactElement {
+}: ConnectionFormPropsWithoutSettings): React.ReactElement {
   const [advancedOpen, setAdvancedOpen] = useState(false);
   const isDarkMode = useDarkMode();
   const isMultiConnectionEnabled = usePreference(
```

Comment:
Are we intending to remove the multi-connection enabled preference in these changes?

Reply:
sounds good, will do!